public/app/features/alerting/unified/mocks/server/handlers/alertRules.ts

```diff
@@ -118,9 +124,18 @@ export const rulerRuleHandler = () => {
   });
 };
 
+export const historyHandler = () => {
+  return http.get('/api/v1/rules/history', () => {
+    return HttpResponse.json(getHistoryResponse([time_0, time_0, time_plus_30, time_plus_30]));
+  });
+};
+
 const handlers = [
   rulerRulesHandler(),
   getRulerRuleNamespaceHandler(),
+  rulerRuleGroupHandler(),
+  rulerRuleHandler(),
```

Comment:
These are duplicated in the array - think it was from a conflict resolution maybe?

Reply:
pushed de commit removing the duplicates!